sw/device/silicon_creator/manuf/lib/BUILD

```
@@ -518,3 +518,23 @@ opentitan_functest(
         "//sw/device/silicon_creator/lib/sigverify:spx_verify",
     ],
 )
+
+opentitan_functest(
+    name = "manuf_cp_test_lock_functest",
+    srcs = ["//sw/device/silicon_creator/rom/e2e:empty_test.c"],
+    cw310 = cw310_params(
+        bitstream = ":bitstream_otp_ctrl_functest",
+        tags = ["jtag"],
+        test_cmds = _MANUF_LC_TRANSITION_TEST_CMDS,
+    ),
+    data = OPENTITANTOOL_OPENOCD_DATA_DEPS,
+    targets = ["cw310_rom"],
+    test_harness = "//sw/host/tests/manuf/manuf_cp_test_lock:manuf_cp_test_lock",
```

nit: some Bazel syntactic sugar, if the target name matches the package name in the label you can drop the former
```suggestion
    test_harness = "//sw/host/tests/manuf/manuf_cp_test_lock"
```